src/components/FeaturedPromo/FeaturedPromo.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import cx from 'classnames';
+
+import { ImageElement } from 'Image';
+import RichText from 'RichText';
+import Link from 'Link';
+
+type FeaturedPromoProps = {
+  author: string;
+  className?: string;
+  excerpt?: string;
+  imageAlt: string;
+  imageHeight?: string;
+  imageSizes?: string;
+  imageSrc: string;
+  imageSrcSet?: string;
+  imageWidth?: string;
+  title: string;
+  titleAs?: 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
+  topic: string;
+  url: string;
+};
+
+export const FeaturedPromo = ({
+  author,
+  className,
+  excerpt,
```

Comment:
Would be good to standardise props, we're using description in a number of other places